addons/opsportal/1.0.x/opsportal-7.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: opsportal
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: opsportal
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.0-7"
+    appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
+    endpoint.kubeaddons.mesosphere.io/opsportal: /ops/portal/
+    values.chart.helm.kubeaddons.mesosphere.io/opsportal: "https://raw.githubusercontent.com/mesosphere/charts/4155f480571eaf82c64ddd63d3d334b1105d0591/stable/opsportal/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: opsportal
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.2.0
```

Comment:
```suggestion
    version: 0.2.1
```
Is this correct?

Reply:
should it not be `0.2.2` ?
https://github.com/mesosphere/charts/blob/master/stable/opsportal/Chart.yaml#L6

Reply:
no `0.2.3` :trollface: 
IIRC there is no _need_ to update the chart version, if @jr0d didnt intended to bump, thats fine. bumps can happen in follow-up PR. if we'd try to always update to newest chart version, we'd never close these PRs…

Reply:
he actually already bumped to `0.2.0` from `0.1.1`…

Reply:
In this case we do want it to be `0.2.3` https://github.com/mesosphere/charts/pull/409/files#diff-db13871e50c21bdbe363cd3384aa647cL6 to pick up the latest RBAC changes.
cc @jr0d

Reply:
yes, when I made this PR the version was more accurate :)